### ISSUE TITLE
Enhancement parameter passing

### ISSUE THREAD

**Issue by we-sell-bags** [woo-j/OkapiBarcode#98]
can you expand the calling system to make it cleaner from the point of view of  parameters.
something like:
so the options are subclasses of the actual barcode.

OKapiBarcode("#barcode")
  .options({font: "OCR-B"}) // Will affect all barcodes
  .EAN13("1234567890128", {fontSize: 18, textMargin: 0})
  .blank(20) // Create space between the barcodes
  .EAN5("12345", {height: 85, textPosition: "top", fontSize: 16, marginTop: 15})
  .render();

**Comment by gredler** [comment]
This isn't Java, is it? It looks like you're using JavaScript / JSON configuration parameters?

**Comment by we-sell-bags** [comment]
Hi it was an example that can be done the same way in java.

There seems to be no simple examples of how to use a single standard call out, to a single piece of java code that can configure all the requires parameters.

The only example i see is  instigating a specific instance for a barcode, then filling that with data.


yes you can change the root instance but then that means a mass of different routines or steering logic.
Far better if this was in a single place... 
The library, it would negate people having to write masses of steering logic code and also  simplify debugging & maintenance of  the library.

It would also mean it could fit into a single line parameter for something like jasper reports, instead of having to call a class outside of jasper to configure all the options inside the barcode.

Basically i would turn it from a programmer task to a user maintainable task.

**Comment by gredler** [comment]
Please feel free to open a PR with a sample implementation of your idea, but right now I don't see how it would be implemented. I also don't see how an API change would allow non-programmers to use Okapi, when the API is meant to be used by other code.
